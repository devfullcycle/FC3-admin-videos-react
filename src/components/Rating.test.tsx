import { render } from "@testing-library/react";
import { Rating } from "./Rating";

describe("Rating", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<Rating rating="L" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
