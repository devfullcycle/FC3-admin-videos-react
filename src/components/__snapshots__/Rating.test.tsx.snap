// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating should render correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-tmfjyz"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 css-jb1ygz-MuiTypography-root"
    >
      L
    </p>
  </div>
</DocumentFragment>
`;
